CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 0.18.2 (Unreleased)
 
+- Update to v2.6.0 of the AWS Terraform Provider.
```

Comment:
Minor - Given that it looks like we didn't release after the update to v2.5.0, we may be able to just leave off that line of the CHANGELOG?

Reply:
sure :) I will do a similar thing in the azure provider PR as well

Reply:
done!